galaxy.yml

```
@@ -0,0 +1,17 @@
+namespace: "theforeman"
+name: "operations"
+description: Ansible assets to manage Foreman operations such as install, upgrade and maintenance tasks
```

starting a description of an ansible collection with "ansible assets" sounds kinda repetitive?

```suggestion
description: Foreman operation tasks such as install, upgrade and maintenance
```